lib/jmap/core/capability/submission_capability.dart

```diff
@@ -1,19 +1,18 @@
 import 'package:equatable/equatable.dart';
-import 'package:jmap_dart_client/http/converter/unsigned_int_converter.dart';
 import 'package:jmap_dart_client/http/converter/unsigned_int_nullable_converter.dart';
 import 'package:jmap_dart_client/jmap/core/capability/capability_properties.dart';
 import 'package:jmap_dart_client/jmap/core/unsigned_int.dart';
 import 'package:json_annotation/json_annotation.dart';
 
 part 'submission_capability.g.dart';
 
-@UnsignedIntConverter()
-@JsonSerializable()
+@UnsignedIntNullableConverter()
+@JsonSerializable(explicitToJson: true, includeIfNull: false)
 class SubmissionCapability extends CapabilityProperties with EquatableMixin {
-  final UnsignedInt maxDelayedSend;
-  final Set<String> submissionExtensions;
+  final UnsignedInt? maxDelayedSend;
+  final Map<String, List<String>>? submissionExtensions;
```

Comment:
any conflict with the original JMAP?

Reply:
please create the test case for it also

Reply:
According to the JMAP documentation, then we need to change the variable so it is correct. It is also true for CYRUS's json

<img width="798" alt="Screenshot 2023-03-24 at 18 03 38" src="https://user-images.githubusercontent.com/80730648/227505039-a5525408-0425-4255-8037-f20209f890fc.png">